docs/porting/security/psa_targets_in_mbed-os.md

```diff
@@ -0,0 +1,561 @@
+
+#### All supported options
+
+```console
+usage: build_psa_compliance.py [-h] -m
+                               {ARM_MUSCA_A1,ARM_MUSCA_B1,CY8CKIT_064S2_4343W}
+                               [-t {ARMCLANG,GNUARM}] [-v {1,2,3,4,5}]
+                               [-s {CRYPTO,INITIAL_ATTESTATION,PROTECTED_STORAGE,INTERNAL_TRUSTED_STORAGE}]
+                               [-r RANGE] [-i INCLUDE]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -m {ARM_MUSCA_A1,ARM_MUSCA_B1,CY8CKIT_064S2_4343W}, --mcu {ARM_MUSCA_A1,ARM_MUSCA_B1,CY8CKIT_064S2_4343W}
+                        Build for the given MCU
+  -t {ARMCLANG,GNUARM}, --toolchain {ARMCLANG,GNUARM}
+                        Build for the given toolchain (GNUARM)
+  -v {1,2,3,4,5}, --verbose {1,2,3,4,5}
+                        Verbose level of the build (default : 3)
+  -s {CRYPTO,INITIAL_ATTESTATION,PROTECTED_STORAGE,INTERNAL_TRUSTED_STORAGE}, --suite {CRYPTO,INITIAL_ATTESTATION,PROTECTED_STORAGE,INTERNAL_TRUSTED_STORAGE}
+                        Test suite name (default : CRYPTO)
+  -r RANGE, --range RANGE
+                        Test suite range (default : all tests), Format :
+                        'test_start_number;test_end_number'
+  -i INCLUDE, --include INCLUDE
+                        Test include path, Format :
+                        <include_path1>;<include_path2>;...;<include_pathn>
+```
+
+### Building TF-M and creating an Mbed OS pull request
+
+<span class="notes">**Note:** Please ensure that both regression tests and PSA API compliance tests pass before creating an Mbed OS pull request.</span>
+
+To build TF-M and create an Mbed OS pull request:
+
+1. Switch to the `mbed-os-tf-m-regression-tests` directory.
+1. Run:
+
+    ```
+    python3 build_tfm.py -m <new target> -t <toolchain> --commit
+    ```
+
+    The command builds TF-M with the config `ConfigCoreIPC.cmake` ,copies the TF-M binary to the location defined by the target attribute `tfm_delivery_dir`, and commits the changes to Mbed OS.
+1. Switch to the `mbed-os` directory and check the latest commit.
+
+   If everything looks good, push the changes to your fork of Mbed OS.
+1. Create an Mbed OS pull request against [https://github.com/ARMmbed/mbed-os](https://github.com/ARMmbed/mbed-os).
+
+   <!--should we include a link to anything in the contributing guide, or does that not matter within the context of the porting guide?-->
+
+**Tip:** When you run the Python script `build_tfm.py` without any options, TF-M is built for all supported targets and the secure binary (TF-M) for each target is copied to the location defined by the target attribute `tfm_delivery_dir`. When you use the `--commit` option, a new Mbed OS commit is created with the new or modified TF-M binaries and `features/FEATURE_PSA/TARGET_TFM/VERSION.txt`.
+
+Example:<!--isn't both the example and the text above it really just part of step 2 above?-->
```

Comment:
I think the explanation should be in step 2 or before it, not after. And the example should be removed or be before the one with -m and -t. Otherwise this can confuse users - it's two takes on a topic